client/src/components/RelatedItemsAndOutfit/RelatedItemsAndOutfit.jsx

```diff
@@ -1,50 +1,36 @@
 import React from 'react';
 import Carousel from './Carousel.jsx';
-import { productsMock, productInfoMock, productStylesMock, relatedProductsMock } from '../../mockData/productInfoMock.js';
-import { reviewsMock, reviewsMetaMock } from '../../mockData/reviewsMock.js';
 
-class RelatedItemsAndOutfit extends React.Component {
-  // when using top level state, this will be a functional component that takes in
-  // all details about the products via props
-
-  constructor (props) {
-    super(props);
-    this.state = {
-      productInfo: productInfoMock,
-      productStyles: productStylesMock,
-      relatedProducts: relatedProductsMock,
-      reviewsMeta: reviewsMetaMock
-    };
-  }
-
-  render() {
-    return (
-      <section
-        id="related-items-and-outfit"
-        className="related-items-and-outfit"
-        data-testid="related-items-and-outfit"
-      >
-        <div id="related-items">
-          <Carousel
-            product={this.state.productInfo}
-            styles={this.state.productStyles}
-            related={this.state.relatedProducts}
-            reviews={this.state.reviewsMeta}
-            carouselType={'related'}
-          />
-        </div>
-        <div id="your-outfit">
-          <Carousel
-            product={this.state.productInfo}
-            styles={this.state.productStyles}
-            related={this.state.relatedProducts}
-            reviews={this.state.reviewsMeta}
-            carouselType={'outfit'}
-          />
-        </div>
-      </section>
-    );
-  }
-}
+const RelatedItemsAndOutfit = ({ productInfo, productStyles, relatedProducts, reviewsMeta }) => {
+  return (
+    <section
```

Comment:
I'm not as familiar with the section heading and I see that it has accessibility benefits. This article mentions that it should have a heading. The team should have a larger discussion about appropriate tags, alt image tags etc. for accessibility as we are doing audits for that this week. 

This accessibility design blog recommends the following for section tags:
* Don’t use the section element if you just need a hook for styling or scripting.
* Use the section element when your content has a theme.
* Give your section element a heading and associate the two programmatically.

https://www.tpgi.com/using-html5-section-element/

Reply:
Interesting. Maybe it would be better to make the top level a div and then the two carousels sections? Since they are semantically clear what they are and have a heading, etc. The top level RelatedItems component is only separated due to the way they have divided up the sections in the spec. I'll make a ticket for it, thank you